core/auth/src/test/resources/log4j2.properties

```diff
@@ -36,3 +36,6 @@ rootLogger.appenderRef.stdout.ref = ConsoleAppender
 #
 #logger.netty.name = io.netty.handler.logging
 #logger.netty.level = debug
+
+logger.cache.name = software.amazon.awssdk.utils.cache
```

Comment:
Actually, could you comment or remove these added logs? We can reactive them on demand in the future

Reply:
Enabled the logs to debug the test failures, new test is intermittently  failing in CI but succeeds in local setup
More context in this PR https://github.com/aws/aws-sdk-java-v2/pull/5259